src/test/kotlin/com/terraformation/backend/customer/db/UserStoreTest.kt

```diff
@@ -945,4 +970,19 @@ internal class UserStoreTest : DatabaseTest(), RunsAsUser {
       assertThrows<AccessDeniedException> { userStore.updateGlobalRoles(setOf(userId), emptySet()) }
     }
   }
+
+  @Nested
+  inner class FunderUser {
+    @Test
+    fun `createFunderUser happy path`() {
+      val email = "testFunderUser@example.com"
+
+      val funderUser = userStore.createFunderUser(email)
+
+      assertNotNull(funderUser.userId)
+      assertEquals(email.lowercase(), funderUser.email)
+      assertEquals(Instant.EPOCH, funderUser.createdTime)
+      assertEquals(UserType.Funder, funderUser.userType)
```

Comment:
A couple things here.

First, I think messages would be useful here, if only for the not-null assertion. Otherwise an assertion failure message will look something like, "Expected null, got 3" which isn't really self-explanatory; you can't tell that it's a user ID just from the message.

But second, maybe instead of these separate assertions, this would be better as a single `assertEquals()` on the entire user object. If you change `FunderUser` to be a `data class`, Kotlin will generate an `equals` method that should work for this test.